content/en/docs/languages/go/instrumentation.md

```diff
@@ -876,7 +875,158 @@ meterProvider := metric.NewMeterProvider(
 
 ## Logs
 
-The logs API is currently unstable, documentation TBA.
+Logs are distinct from metrics and traces in that **there is no user-facing
+OpenTelemetry logs API**. Instead, there is tooling to bridge logs from existing
+popular log packages (e.g. slog, logrus, zap, logr) into the OpenTelemetry
+ecosystem. For rationale behind this design decision, see
+[Logging specification](/docs/specs/otel/logs/).
+
+The two typical workflows discussed below each cater to different application
+requirements.
+
+### Direct to collector (Experimental)
+
+In the direct to collector workflow, logs are emitted directly from an
```

Comment:
```suggestion
In the direct-to-Collector workflow, logs are emitted directly from an
```

Reply:
Same is in Java. Should I fix it there as well?

Reply:
Shouldn't the header be also named "direct-to-Collector", then?

Reply:
Yes, there should be something consistent